how-to/create-working-group.md

```diff
@@ -0,0 +1,9 @@
+This document describes some part of how to create a working group in the InnerSource Commons.
+
+* Create a _Slack_ channel with the name of your working group.
+* Send out invites using the [InnerSource Commons calendar].
+* Schedule video calls with https://meet.jit.si/.
+* Make a folder with the name of your working group in the [InnerSource Commons GDrive].
+
+[InnerSource Commons calendar]: https://github.com/InnerSourceCommons/foundation/blob/master/members/calendar.md
```

Comment:
Non-Members don't have access to the foundation repo.
Maybe port this info over to foundation-governance?